api-docs/general-api-info/svc-catalog-ovw.rst

```diff
@@ -47,7 +47,7 @@ documentation`_.
 |                        | who request access to Rackspace Cloud service      |
 |                        | access to Rackspace Cloud services.                |
 |                        |                                                    |
-|                        | Cloud Identity is the name of the Rackspace Cloud  |
+|                        | Identity is the name of the Rackspace Cloud        |
 |                        | service of this type. The Rackspace service is     |
```

Comment:
Should this be "The Rackspace Identity service is..."

Reply:
Fixed